src/modules/projects/components/tables/ProjectEnrollmentsTable.tsx

```diff
@@ -80,9 +65,9 @@ const ProjectEnrollmentsTable = ({
                 )}
               />
             </Grid>
-          )}
+          }
           <Grid item flexGrow={1}>
-            {searchable ? (
+            {true ? (
```

Comment:
Could this be simplified? Remove `true` and always render <ClientSearchInput>?

Reply:
oops good catch I didn't mean to leave that! I was just testing. will fix!